src/unix/os390.c

```diff
@@ -830,6 +857,7 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
       ep = loop->ep;
       if (pe->is_msg) {
         os390_message_queue_handler(ep);
+        nevents++;
```

Comment:
This seems like an unrelated fix? If so, could you make a separate PR for just this line

Reply:
Sorry for the confusion, this fixes a bug that cause `uv__io_poll()` to hang if `uv__make_close_pending()` is deferred. The problem is that if `nevents` doesn't get incremented here, then it will be 0 when it reaches the `if (nevents != 0)` check at the bottom of the loop and won't enter the branch to return from the function. So it will loop back to `epoll_wait()` even though there are no more events remaining, and block indefinitely. So this fix needs to go together with the PR to resolve the #3601 test failure.